File: API/Dms.Domain/Entities/Tip.cs
using Dms.Domain.Common;

namespace Dms.Domain.Entities
{
    public class Tip : BaseEntity
    {
        public string Title { get; set; } = string.Empty;
        public string ShortDescription { get; set; } = string.Empty;
        public string Content { get; set; } = string.Empty;
        public string? ImageUrl { get; set; }
        public string Author { get; set; } = "Kỹ thuật viên DMS";
        public bool IsActive { get; set; } = true;
    }
}

File: my-app/src/types/tip.ts
export interface Tip {
  id: string;
  title: string;
  slug?: string;
  shortDescription: string;
  content: string;
  imageUrl?: string;
  author: string;
  isActive: boolean;
  created?: string;
}

export interface CreateTipInput {
  title: string;
  slug?: string;
  shortDescription: string;
  content: string;
  imageUrl?: string;
  author: string;
}
